src/components/layout/BaseFooter.astro

```diff
@@ -1,6 +1,11 @@
 ---
 ---
-
+       <button class="backToTop">
+        <div class="backToTopImgWrap">
+            <img src="assets/images/up.png" alt="" class="backFirstImg">
+            <img src="assets/images/up.png" alt="" class="backLastImg">
+        </div>
+    <script src="assets/js/index.js"></script>    
```

Comment:
Instead of using the `assets/js/index.js` import here. I would prefer to include the script here as a `<script is:inline>`. Also, Since the file is located in `/src/js/index.js`, this won't work.

Reply:
could you maybe modify this and make necessary changes? please.

Reply:
so like this?

Reply:
should probably kill image alltogether and just use svg from https://heroicons.com